<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menProject">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <widget class="QMenu" name="menuProjectRecent">
     <property name="title">
      <string>Recent</string>
     </property>
    </widget>
    <addaction name="actionProjectNew"/>
    <addaction name="separator"/>
    <addaction name="actionProjectLoad"/>
    <addaction name="menuProjectRecent"/>
    <addaction name="separator"/>
    <addaction name="actionProjectSave"/>
    <addaction name="actionProjectSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionProjectPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionProjectExit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuWindowViewMode">
     <property name="title">
      <string>View mode</string>
     </property>
     <addaction name="actionWindowViewModeWindowed"/>
     <addaction name="actionWindowViewModeTabbed"/>
    </widget>
    <addaction name="actionWindowNext"/>
    <addaction name="actionWindowPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionWindowTile"/>
    <addaction name="actionWindowCascade"/>
    <addaction name="separator"/>
    <addaction name="actionWindowClose"/>
    <addaction name="actionWindowCloseAll"/>
    <addaction name="separator"/>
    <addaction name="menuWindowViewMode"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout"/>
    <addaction name="actionHelpAboutQt"/>
   </widget>
   <addaction name="menProject"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionProjectNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionProjectLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionProjectSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionProjectSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionProjectExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionWindowNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionWindowPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionWindowTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionWindowCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionWindowClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionWindowCloseAll">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionHelpAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionProjectPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionNo_recent_files">
   <property name="text">
    <string>No recent files</string>
   </property>
  </action>
  <action name="actionWindowViewModeWindowed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Windowed</string>
   </property>
  </action>
  <action name="actionWindowViewModeTabbed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tabbed</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
